.ci/scripts/run-gradle-profiler.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# drop page cache and kernel slab objects on linux
+[[ -x /usr/local/sbin/drop-caches ]] && sudo /usr/local/sbin/drop-caches
+rm -Rfv $WORKSPACE/gradle-user-home
+mkdir -p $WORKSPACE/gradle-user-home/init.d && cp -v $WORKSPACE/.ci/init.gradle $WORKSPACE/gradle-user-home
```

Comment:
We configure the max worker here passed from the profiler to the gradle invocation

Reply:
At the end comparing multiple benchmarks among each other doesn't make sense IMO. What we're interested here is just a comparison within the benchmark that is done in one run to see a tendency. I'd keep it to make as efficient as possible.

Reply:
> At the end comparing multiple benchmarks among each other doesn't make sense IMO. What we're interested here is just a comparison within the benchmark that is done in one run to see a tendency. I'd keep it to make as efficient as possible.

Gotcha, and the "run" just compares HEAD against the PR changes so you get an idea if your change introduces a regression (or improvement), yes?

Reply:
exactly